### ISSUE TITLE
Broken time column rendering with 35c3 schedule

### ISSUE THREAD

**Issue by johnjohndoe** [EventFahrplan/EventFahrplan#93]
# Environment
- App version: v.1.37.0, v.1.38.0
- Branch: `master`
- Device: not relevant
- Schedule: Configure [35c3 everything.schedule.xml snapshot](https://github.com/voc/35C3_schedule/raw/b72d21c1042af37f9da09428e1225e18e4502db2/everything.schedule.xml)

# How to reproduce
- Start the app
- Navigate to day 2 or day 3 from the top menu
- Pan to the room `OIO Vortrags-Arena` to see the earliest event

# Observed behavior
- The time column view on the left side only shows time entries from `00:00` till `02:15`. Below the time text is not rendered as shown in the screenshot:

![35c3-schedule-day2](https://user-images.githubusercontent.com/144518/50336072-85433a00-050d-11e9-952e-a80b0535a04e.png)

# Expected behavior
- The time column view on the left side shows the time entries in a regular series 15 minutes segments. Compare with day 1 or day 4.

# Intermediate research
- The rendering takes place here:
https://github.com/EventFahrplan/EventFahrplan/blob/545c22eadc54951d75c0bd00641c7e3660a3dcde/app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java#L496-L524
- A quick debugging session revealed that the `while` loop stops because the `Conference` object does not satisfy the condition due to `conference.getLastEventEndsAt()`. See [`Conference#calculateTimeFrame`](https://github.com/EventFahrplan/EventFahrplan/blob/545c22eadc54951d75c0bd00641c7e3660a3dcde/app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/Conference.kt#L24). Further investigation is needed.

# Related
- [Issue #61: Broken time column rendering with event time of 00:00:00](https://github.com/EventFahrplan/EventFahrplan/issues/61)

**Comment by poschi3** [comment]
I investigated some time in this issue and found out, that for day 2 in the calculateTimeFrame method the "firstEvent" is "Geschichten aus der Cobol Gruft" from Chaos West. That's a bit strange, because regarding to there website the time slot is on [day 1 from 23:00–23:50](https://fahrplan.chaos-west.de/35c3chaoswest/talk/GBVHJT/).

I've searched for the event in the view and found out, that EventFahrplan put the event on day 2 from 0:00 to 0:50. So everything from Choas West is of by one hour.
![2018-12-23 20 11 26-2](https://user-images.githubusercontent.com/825911/50386826-086fb600-06ef-11e9-84e7-68dd37c280fe.jpg)
